schemas/delta.json

```diff
@@ -64,6 +79,27 @@
                 }
               }
             }
+          },
+          "meta": {
+            "type": "array",
+            "items": {
+              "type": "object",
+              "required": [
+                "path",
+                "value"
+              ],
+              "properties": {
+                "path": {
+                  "type": "string",
+                  "description": "The local path to the data value",
+                  "example": "navigation.courseOverGroundMagnetic"
+                },
+                "value": {
+                  "$ref": "./definitions.json#/definitions/meta",
+                  "additionalProperties": true
```

Comment:
I don't think there is a need for additionalProperties here. If so an explanation is needed.

Reply:
see PR #491 removing additionalProperties: true